_posts/2024-10-14-2025-CFP.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: post
+category: post
+title: "BSidesSF 2025 Call for Presentations is open!"
+date: 2024-10-14
+---
+
+Planning for BSidesSF 2025 is progressing on schedule.
+
+You will all be excited to know that we have opened up the CFP.
+[Read all about it here](/cfp)
+
+Additionally, we are always looking for more sponsors. If you are 
+interested check out the [sponsors benefits](/sponsors/benefits) 
+and reach out to us at <a href="mailto:sponsors[at]bsidessf.org">sponsors[at]bsidessf.org</a>
```

Comment:
```suggestion
and reach out to us at sponsors@bsidessf.org.
```

Reply:
You dislike the anchor tag for emails?

Reply:
We've just used the real email in past blog posts. Let's not make it complicated for sponsors.

Reply:
Gotcha, using email address only.